parser-typechecker/src/Unison/Util/ColorText.hs

```diff
@@ -82,7 +92,16 @@ toANSI (AnnotatedText chunks) =
       )
   resetANSI = pure . ANSI.setSGRCode $ [ANSI.Reset]
   toANSI :: Color -> Seq String
-  toANSI c = pure . ANSI.setSGRCode $ case c of
+  toANSI c = pure $ ANSI.setSGRCode (toANSI' c)
+
+  toANSI' :: Color -> [ANSI.SGR]
+  toANSI' c = case c of
+    Default  -> []
+    Background c c2 -> (setBg <$> toANSI' c) <> toANSI' c2 where
+      setBg (ANSI.SetColor ANSI.Foreground intensity color) =
+             ANSI.SetColor ANSI.Background intensity color
+      setBg sgr = sgr
+    Invert c -> [ANSI.SetSwapForegroundBackground True] <> toANSI' c
```

Comment:
This change allows text that is both bold and underlined, or other combinations of ANSI terminal formatting.